cli/command/container/opts.go

```diff
@@ -899,3 +921,12 @@ func validateAttach(val string) (string, error) {
 	}
 	return val, errors.Errorf("valid streams are STDIN, STDOUT and STDERR")
 }
+
+func convertToNetworkmode(netmode opts.NetworkOpt) container.NetworkMode {
+	networkIDOrName := "default"
+	for _, net := range netmode.Value() {
+		networkIDOrName = net.Target
+		break
```

Comment:
I would instead write this as:

```go
netOptVal := netmode.Value()
if len(netOptVal) > 0 {
        networkIDOrName = netOptVal[0].Target
}
```

But is it correct to silently ignore everything beyond the first value?